xarray/core/variable.py

```diff
@@ -2432,42 +2437,6 @@ def concat(cls, variables, dim="concat_dim", positions=None, shortcut=False):
 
         return cls(first_var.dims, data, attrs)
 
-    def copy(self, deep=True, data=None):
```

Comment:
shall we keep this but use `self.__copy__` under the hood?

It lines up well with `Dataset.copy()` and `DataArray.copy()`